Features/AdminCommands/UnBanCommand.cs

```diff
@@ -0,0 +1,37 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.ContextChecks;
+using ArtcordAdminBot.Features.Helpers;
+
+namespace ArtcordAdminBot.Features.AdminCommands
+{
+    public partial class AdminCommandGroup
+    {
+        
+
+        [Command("Unban")]
+        [RequirePermissions(DiscordPermissions.BanMembers)]
+        public async Task BanAsync(CommandContext ctx, 
+        [System.ComponentModel.Description("The user to unban.")]ulong targetUser, 
+        [System.ComponentModel.Description("The reason for the unban.")] string? reason = null)
+        {
+            
+            // Place DB logic here for unbans
+            //await _databaseService.NewBanRecordAsync();
+
+            await ctx.Guild!.UnbanMemberAsync(targetUser, reason);
+
+            var embed = new DiscordEmbedBuilder
+            {
+                Title = "User UnBanned",
```

Comment:
Not a big deal, but the pascal case ("UnBanned") just feels odd xd

Reply:
i thought i changed aalla of those to Unbanned